shaw-virtual-desktop/base/kustomization.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: shaw-virtual-desktop
+commonLabels:
+  app: shaw-virtual-desktop
+resources:
+- deployments/postgres.yaml
+- volumes/postgres-data.yaml
+- services/postgres.yaml
+- services/guacamole.yaml
+- deployments/guacamole.yaml
+- routes/shaw-virtual-desktop.yaml
+- deployments/redis.yaml
+- services/oauth2-proxy.yaml
+- services/redis.yaml
+- volumes/redis-data.yaml
+- deployments/guacd.yaml
+- services/guacd.yaml
+- volumes/shared-drives.yaml
```

Comment:
if you could sort the resources alphabetically, that'd be cool.

Reply:
Will do.

Reply:
Sorted!